azurefunctions-extension/pom.xml

```diff
@@ -0,0 +1,257 @@
+<!--
```

Comment:
We done need this pom.xml anymore since we are using gradle.

Reply:
Yes, fixed!.